order_bot/index.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+const request = require('request');
+const faker = require('faker');
+
+// Hits variable controls how many times the request loop will autoamtically run
+let hits = 900;
+
+// Automated request loop
+let i;
+for(i = 0; i < hits; i++) {
+  /**
+   * Using Faker to produce fake user information
+   * Faker API documentation at https://github.com/marak/faker.js
+   */
+  const payload = {
+    order: {
+      units: [
+        {
+        sku: 'a5296ab9-9eee-7ba0-0a79-b801594f2c91',
+        quantity: faker.random.number()},
+        {
+        sku: 'a5296ab9-9eee-7ba0-0a79-b801594f2c92',
+        quantity: faker.random.number()}
+      ],
+      recipients: {
+        name: faker.name.findName(),
+        address: {
+          street: faker.address.streetAddress(),
+          city: faker.address.city(),
+          state: faker.address.stateAbbr(),
+          zip: faker.address.zipCode()
+        },
+        phone: faker.phone.phoneNumberFormat(),
+        email: faker.internet.email()
+      }
+    }
+  };
+
+/**
+ * HTTP PUT Request hitting endpoint /order
+ * Converting payload to string for readibility purposes
+ */
+  request({
+    uri: 'http://localhost:3000/order',
```

Comment:
Don't hardcode IP Addresses, host names, or .coms into your files, even statically. Set them up to use a variable, so that we can have stronger control over it moving forward.

Localhost is not the name of the server we will be deploying to.